### PR TITLE
Upgrade airbrake version

### PR DESCRIPTION
**Pull request by ostap0207**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ Options:
 
 * __apiKey:__ Valid Airbrake API Key (required)
 * __host:__ Your host, to be displayed in Airbrake. (default: require('os').hostname())
+* __env:__ Environment, to be displayed in Airbrake. (default: production)
```
Comment (urmastalimaa) [inline README.md]: Why you would want to default to production?

Comment (ostap0207) [inline README.md]: I think explained in the commit message. To keep current version backward compatible.

Comment (urmastalimaa) [inline README.md]: I did not quite understand the commit message, do you mean that previously errors from every env were added in newer airbrake versions dev and test are ignored? Maybe you can clarify the commit message a bit.

Comment (ostap0207) [inline README.md]: Will do. But yeah, as there was no option to set the env it was 'development' by default and in new version it's ignored.

**Comment by urmastalimaa** [comment]
+1

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "winston-airbrake",
-  "version": "0.0.3",
+  "version": "0.1.0",
```
Comment (willmore) [inline package.json]: You should clarify in the commit message why you increment minor version.

Comment (ostap0207) [inline package.json]: Updated

**Comment by indrekj** [comment]
Does it maybe make sense to try to follow airbrake-ruby options: https://github.com/airbrake/airbrake-ruby#ignore_environments

**Comment by indrekj** [comment]
or whatever airbrake-js is using

**Comment by ostap0207** [comment]
node-airbrak has 'env', so it's inline with it.

**Comment by ostap0207** [comment]
Closing as was merged in original repository.

**Pull request closed without merging**